tox.ini

```diff
@@ -89,4 +89,4 @@ whitelist_externals = bash
 skip_install = True
 deps =
     flake8
```

Comment:
Also this `flake8` should be pinned to 3.4.1 since py27-lint-docstring-include-list is one of the mandatory Travis build jobs.